org.eclipse.jdt.core.javac/src/org/eclipse/jdt/internal/javac/JavacCompiler.java

```diff
@@ -179,6 +179,9 @@ public int errorCount() {
 					return this.isInGeneration ? 0 : super.errorCount();
 				}
 			};
+			// https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/issues/955
+			// enable annotation processing
+			javacOptions.put(Option.A, "");
```

Comment:
This is more or less already done in JavacUtils lines 183..186. However, JDT mandates that annotation processing is enabled to consider it. Maybe the issue is just that the necessary JDT flags were missing?

Reply:
You are right. I have fixed JavacUtils lines 183..186.